README.md

```diff
@@ -227,9 +227,9 @@ const pubNubClient = new PubNub(pubNubConfig.Demo.keySet);
 const App = () => {
   <PubNubProvider client={pubNubClient}>
     <PubNubConsumer>
-      {client => "success!" /* do something now */ }
+      {client => 'success!' /* do something now */}
     </PubNubConsumer>
-  </PubNubProvider>
+  </PubNubProvider>;
```

Comment:
Semi-colon seems odd here

Reply:
This was incorrect BTW, and my prettier plugin added the semicolon here. I have fixed the example and it put the parentheses, so it should be fine now.